Bitget.Net/Converters/OnOffConverter.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Bitget.Net.Converters
+{
+    /// <inheritdoc />
+    public class OnOffConverter : JsonConverter<bool>
```

Comment:
These converters should not be necessary. The base library should already convert On/Off and Yes/No to the correct bool values when using the default configuration.